themes/default/layouts/registry/how-to-guide.html

```diff
@@ -1,11 +1,12 @@
 {{ define "main" }}
+{{ $lastUrlSlug := index (last 1 (split (delimit (split $.Page.File "/") "," "") ",")) 0 }}
```

Comment:
Is this needed? or maybe a rebase leftover?

Reply:
yes thank you so much. will remove!